### PR TITLE
OHC collectionobject module

### LINKED ISSUE #158: Converter module for Cataloging OHC
At this time, this work will include support for fields included in the OHC remains collectionobject data in the migration queue that:

- do not exist in collectionobjects_common, collectionobjects_anthro, collectionobjects_annotation, or collectionobjects_nagpra;
- or are that are in these namespaces but are overridden in OHC extension 

This module will initially include the listed fields in the following namespaces: 

- ns2:collectionobjects_ohc
  - descriptionLevel
  - namedTimePeriod
  - oaiCollectionPlace

This module will handle overrides to the following fields:
- assocPeople (source: conceptauthorities:name(ethculture))
- objectName (source: conceptauthorities:name(nomenclature))

### PR DESCRIPTION
**Pull request by kspurgin**
Fixes #158 (eventually)

### CODE REVIEW

**Comment by kspurgin** [comment]
@mark-cooper I think this is ready to go... Not sure what to do about any config yaml files, though.

File: lib/collectionspace/converter/ohc/collectionobject.rb [hunk 1]
```diff
@@ -0,0 +1,139 @@
...
+            'assocpeoplenote',
+            'objectnametype',
+            'objectnamesystem',
+            'objectname',
+            'objectnamecurrency',
+            'objectnamenote',
+            'objectnamelevel',
+            'objectnamelanguage'
+          ]
+          safe = attributes.clone
```
Comment (kspurgin) [inline lib/collectionspace/converter/ohc/collectionobject.rb]: @mark-cooper -- So, if I did not use '.clone' here, `safe.delete(fieldname)` also affected the main attributes hash. Attributes I needed to send through the special OHC overrides were no longer there. 

I knew I'd ascertained the necessity of `.clone` in some situations... Do you know off the top of your head what the difference it here, as opposed to the copy of the string in the DateParser?

**Comment by kspurgin** [comment]
@mark-cooper If this looks good to you, and you merge it, then it's time for #159 -- the integrating it all in the tool step.

File: lib/collectionspace/converter/ohc/collectionobject.rb (hunk 1 above)
Comment (mark-cooper) [reply lib/collectionspace/converter/ohc/collectionobject.rb]: @kspurgin I believe it's just implementation differences for data structures that is generally in line with how other languages work. For standard data types (ints, strings etc.) there's generally copy type behavior and for data structures (array, hash etc.) there's generally reference type behavior (even though for Ruby everything is an object behind the scenes).

**Pull request merged**